src/cache.c

```diff
@@ -86,15 +87,16 @@ ts_cache_invalidate(Cache *cache)
  * Each call to cache_pin MUST BE paired with a call to cache_release.
  *
  */
-extern Cache *
+extern TSDLLEXPORT Cache *
```

Comment:
`extern TSDLLEXPORT` are only used in header files, while they are omitted in the source:
```suggestion
Cache *
```

Reply:
I believe I've removed all TSDLLEXPORTs in C files for multinode commits